momentum_scanner_intraday.py

```diff
@@ -57,6 +62,39 @@ def get_risk_query(token, stoploss=0.05, profittaking=0.05):
     print("enter at:", price, "stop-loss:", slprice, "profit-taking:", ptprice)
     return slprice, ptprice
 
+def find_chain(dex):
```

Comment:
And then this function can just be lookups.

Reply:
Done